tests/pom.xml

```diff
@@ -85,9 +88,9 @@
             <version>2.19.1</version>
             <configuration>
               <skipTests>false</skipTests>
-              <includes>
-                <include>%regex[.*LocalIntegrationTest.*]</include>
-              </includes>
+              <systemPropertyVariables>
+                <test.local>true</test.local>

```

Comment:
Can we invert this, and actually have the user specify execution mode through the system property?
```
mvn verify -Dtest.mode=local
mvn verify -Dtest.mode=remote
```
and then activate the profiles based on the property.
```XML
    <activation>
      <property>
        <name>test.mode</name>
        <value>local</value>
      </property>
    </activation>
```